ajax/libs/vue-clickaway/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "vue-clickaway",
+  "description": "Reusable clickaway directive for reusable Vue.js components",
+  "version": "2.1.0",
+  "filename": "vue-clickaway.min.js",
+  "author": {
+    "name": "Denis Karabaza",
+    "email": "denis.karabaza@gmail.com"
+  },
```

Comment:
It's better to be the same as the upstream. https://github.com/simplesmiler/vue-clickaway/blob/2.1.0/package.json#L5